MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,471 @@
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+    
+    using Syncfusion.Maui.Core.Hosting;
+
+    public static class MauiProgram
+    {
+	    public static MauiApp CreateMauiApp()
+	    {
+	        var builder = MauiApp.CreateBuilder();
+		    builder
+			    .ConfigureSyncfusionCore()
+			    .UseMauiApp<App>()
+			    .ConfigureFonts(fonts =>
+			    {
+				    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+				    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+			    });
+
+		    return builder.Build();
+	    }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Creating data object
+
+The `SfDataForm` is a data edit control so, create a data object to edit the data object.
+
+Here, the data object named **DataModel** created with some properties.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+
+    public class DataModel
+    {
+        public string FirstName { get; set; }
+    
+        public string MiddleName { get; set; }
+    
+        public string LastName { get; set; }
+    
+        public string ContactNumber { get; set; }
+    
+        public string Email { get; set; }
+   
+        public string Address { get; set; }
+
+        public DateTime? BirthDate { get; set; }
+    
+        public string GroupName { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+Create a model repository class with DataModel property initialized with required data in a new class file as shown in the following code example and save it ViewModel.cs file:
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public class ViewModel
+    {
+        public DataModel DataModel {get; set;}
+        
+        public ViewModel()
+        {
+            this.DataModel = new DataModel();
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting data object
+
+To populate the labels and editors in the data form, set the `DataObject` property.
+
+{% tabs %}
+{% highlight MainPage.xaml %}
+
+    <?xml version="1.0" encoding="utf-8" ?>
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+                x:Class="GettingStarted.MainPage">
+
+                    <ContentPage.BindingContext>
+                        <local:ViewModel/>
+                    </ContentPage.BindingContext>    
+	                        <dataForm:SfDataForm x:Name="dataForm" 
+						    DataObject="{Binding DataModel}"/>    
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.DataObject = new DataModel();
+
+{% endhighlight %}
+{% endtabs %}
+
+Now, run the application to render the `data form` to edit the data object as in the following screenshot:
```

Comment:
Remove this

Now, run the application to render the `data form` to edit the data object as in the following screenshot:

Reply:
above mentioned content removed